src/plugins/index_pattern_field_editor/public/components/field_editor/form_fields/format_field.tsx

```diff
@@ -8,12 +8,31 @@
 import React from 'react';
 
 import { UseField } from '../../../shared_imports';
+import { FormatSelectEditor, FormatSelectEditorProps } from '../../field_format_editor';
 
-export const FormatField = () => {
+export const FormatField = ({
+  fieldAttrs,
+  indexPattern,
+  fieldFormatEditors,
+  fieldFormats,
+  uiSettings,
+  onChange,
+  onError,
+}: FormatSelectEditorProps) => {
   return (
     <UseField path="format">
       {({ value, setValue }) => {
-        return <div>Format editor</div>;
+        return (
```

Comment:
Apart from `value` and `setValue` you also have a `setErrors`. You can use it to provide any errors on the "format" field.

```js
<UseField path="format">
      {({ value, setValue, setErrors }) => {
        const onError = (errors) => {
	      // The form field errors is an Array of translated string
  		  const errorsToString: string[] = errors.map(...whatever is needed);
  		  setErrors(errorsToString); 
		};

        return (
          <FormatSelectEditor
            fieldAttrs={fieldAttrs}
            indexPattern={indexPattern}
            fieldFormatEditors={fieldFormatEditors}
            fieldFormats={fieldFormats}
            uiSettings={uiSettings}
            onChange={setValue}
            onError={onError}
          />
        );
      }}
</UseField>
```